Dizzy/Screens/Login/AppInfoView.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  AppInfoView.swift
+//  Dizzy
+//
+//  Created by stas berkman on 05/06/2019.
+//  Copyright © 2019 Dizzy. All rights reserved.
+//
+
+import UIKit
+
+protocol AppInfoViewDelegate: class {
+    func aboutButtonPressed()
+    func termsOfUsButtonPressed()
+    func privacyPolicyButtonPressed()
+    func contactUsButtonPressed()
+}
+
+class AppInfoView: UIView {
+
+    weak var delegate: AppInfoViewDelegate?
+
+    let stackView: UIStackView = UIStackView()
+    
+    let aboutButton: UIButton = UIButton()
+    let aboutSeparatorView: UIView = UIView()
+    
+    let termsOfUseButton: UIButton = UIButton()
+    let termsOfUseSeparatorView: UIView = UIView()
+
+    let privacyPolicyButton: UIButton = UIButton()
+    let privacyPolicySeparatorView: UIView = UIView()
+
+    let contactUsButton: UIButton = UIButton()
+    
+    init() {
+        super.init(frame: CGRect.zero)
+        addSubviews()
+        layoutViews()
+        setupViews()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func addSubviews() {
+        
+        stackView.addArrangedSubview(aboutButton)
+        stackView.addArrangedSubview(aboutSeparatorView)
+        stackView.addArrangedSubview(termsOfUseButton)
+        stackView.addArrangedSubview(termsOfUseSeparatorView)
+        stackView.addArrangedSubview(privacyPolicyButton)
+        stackView.addArrangedSubview(privacyPolicySeparatorView)
+        stackView.addArrangedSubview(contactUsButton)
+        
+        self.addSubview(stackView)
+    }
+    
+    private func layoutViews() {
+        layoutStackView()
+        layoutAboutSeparatorView()
+        layoutTermsOfUseSeparatorView()
+        layoutPrivacyPolicySeparatorView()
+    }
+    
+    private func layoutStackView() {
+        stackView.snp.makeConstraints { stackView in
+            stackView.top.leading.trailing.bottom.equalToSuperview()
+        }
+    }
+    
+    private func layoutAboutSeparatorView() {
+        aboutSeparatorView.snp.makeConstraints { aboutSeparatorView in
+            aboutSeparatorView.height.equalTo(0.5)
+            aboutSeparatorView.width.equalToSuperview().multipliedBy(0.6)
+        }
+    }
+    
+    private func layoutTermsOfUseSeparatorView() {
+        termsOfUseSeparatorView.snp.makeConstraints { termsOfUseSeparatorView in
+            termsOfUseSeparatorView.height.equalTo(0.5)
+            termsOfUseSeparatorView.width.equalToSuperview().multipliedBy(0.6)
+        }
+    }
+    private func layoutPrivacyPolicySeparatorView() {
+        privacyPolicySeparatorView.snp.makeConstraints { privacyPolicySeparatorView in
+            privacyPolicySeparatorView.height.equalTo(0.5)
+            privacyPolicySeparatorView.width.equalToSuperview().multipliedBy(0.6)
+        }
+    }
```

Comment:
please make new class for the button and separator and use it, you have a lot of duplicate code and this will solve it.
more than that  the 0.5 and 0.6 will need to be save as class variables

Reply:
Done!